be/src/exprs/vectorized/runtime_filter.h

```diff
@@ -621,6 +621,33 @@ class RuntimeBloomFilter final : public JoinRuntimeFilter {
         return true;
     }
 
+    // filter zonemap by evaluating _min between min_column / max_column
+    // or _max between min_column / max_column
+    bool filter_zonemap_with_min_max(const Column* min_column, const Column* max_column) const {
```

Comment:
suggest to use scalar types in this function argument: std::optional<CppType> min, std::optional<CppType> max;
decomposing a scalar value from the single-item column of the single-column chunk can be write as utilitly function in ColumnHelper.

Reply:
OK. Will fix that.